projects/aca-content/folder-rules/src/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -82,6 +88,9 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnCh
     this._parameterConstraints = value.map((obj) => ({ ...obj, constraints: this.parseConstraintsToSelectOptions(obj.constraints) }));
   }
 
+  private readonly tagsRelatedPropertiesAndAspects = ['cm:tagscope', 'cm:tagScopeCache', 'cm:taggable'];
+  private readonly categoriesRelatedPropertiesAndAspects = ['cm:categories', 'cm:generalclassifiable'];
```

Comment:
Does `cm:generalclassifiable` aspect affect only categories? Can we verify it with someone from BE?

Reply:
sure I will try to ask somebody from BE